### ISSUE TITLE
Background color and logo takes time to load in login page

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#6136]
**Bug description:**
background color and logo takes time to load

**Steps to reproduce:**
1. Go to https://login.niue.opencrvs.org/?lang=en
2. Observe 

**Actual result:**
background color and logo takes time to load

**Expected result:**
Should be loading faster

**Screen recording:**

https://github.com/opencrvs/opencrvs-core/assets/94350336/bc7b6a1c-2cb6-418a-9024-2bcdcc07355d

**Tested on:**
https://login.niue.opencrvs.org/?lang=en

**Comment by euanmillar** [comment]
Changing the logo via the system UI had no effect on the login app unless the user cleared cache.  This is because the login app loaded the image from localStorage in 1.3.  Instead I put it in localForage so that logo updates from the UI work without a cache clear.  I did it in this PR based on Niue [reqs](https://github.com/opencrvs/opencrvs-core/pull/6016/commits/0b941c0ebcab345b56287d814fbc321cd3852cab).  The unintended consequence is that it takes a few seconds to load the image each time from IndexedDB. We ran out of time to address it.

**Comment by jpye-finch** [comment]
@Zangetsu101 @tahmidrahman-dsi @euanmillar 

Some of the changes to how the Login app loads done in 1.5. Is it possible they would have fixed this bug?

**Comment by Zangetsu101** [comment]
@jpye-finch yes https://github.com/opencrvs/opencrvs-core/pull/7010 should have fixed this issue already as we now show the loading spinner until everything's loaded properly

**Comment by SyedaAfrida** [comment]
@Zangetsu101 Fixed in Core  V1.5.0


https://github.com/opencrvs/opencrvs-core/assets/94350336/817e59f3-59cd-40b2-aea5-33597c401c5d
